### DISCUSSION TITLE
Can Adobe Creative Cloud be included here?

### DISCUSSION THREAD

**Discussion by DMahalko** [microsoft/winget-pkgs#61920] [Q&A] [answered]
Adobe CC is such a headache to keep up to date. Their own install packages are ridiculously huge, usually 2+ gigs per compressed application package, which itself is a huge ball of separate sub-packages.

Can it be included here so that I don't have to be constantly generating new update packages by my lone self and maintenance can be a collaborative effort?

Ref: Deploy Adobe packages using Intune:
https://helpx.adobe.com/uk/enterprise/kb/deploy-packages-using-ms-intune.html

**Comment by licon4812** [comment, marked as answer]
I tried adding it. The main issue with CC is that the installer sha changes every time the install URL is hit
